officers.json

```diff
@@ -41,6 +40,20 @@
     "term_end": "SPRING_2019",
     "graduating_semester": "SPRING_2021"
   },
+  {
+    "name": "Joshua Castor",
+    "position": "Technical Officer",
```

Comment:
The convention for position titles is that they're all caps, with spaces replaced by underscores, so we should probably standardize this as "TECHNICAL_OFFICER" and "SECRETARY"